scripts/prepare_sector_network.py

```diff
@@ -461,7 +461,7 @@ def add_biomass(n, costs):
         nodes + " biomass EOP",
         bus0=spatial.biomass.nodes,
         bus1=nodes,
-        bus2="co2 atmosphere",
+        # bus2="co2 atmosphere",

```

Comment:
why is this kicked out?

Reply:
upon further research, I think we should not count the emissions from biomass as long as we set a "sustainable" biomass potential values. It is assumed that biomass emissions will be reabsorbed by forests/trees planted to replace those cut down to burn.

Reply:
This is also the same approach done in PyPSA-Eur